README.md

```diff
@@ -1,17 +1,192 @@
+```
+python train.py --config_file ./configs/resV1c50_32x_dec8x_tr2+2_512x512_adamW_step_640k_ade20k.py
+```
+
+But, it got the ```CUDA error: an illegal memory access was encountered``` when trained with multi-gpu (```norm_cfg = dict(type='SyncBN', requires_grad=True)```):
+
+```
+python -m torch.distributed.launch --nproc_per_node=8 --master_port=29500 train.py  --launcher pytorch \
+--config_file ./configs/resV1c50_32x_dec8x_tr2+2_512x512_adamW_step_640k_ade20k.py
+``` -->
+
+
+### Evaluation
+<!-- 1. directly resize image and ground truth to 512x512 by bilinear interpolation and nearest interpolation, respectively. ('test_mode': 'direct_resize')
+
+The default is for single-gpu evaluation with batch size being 1.
+```
+python evaluate.py --eval \
+--config_file configs/resV1c50_segtr2_512x512_ade20k.yaml \
+--checkpoint log/checkpoint_0131.pth.tar \
+--data_root /path/to/ADEChallengeData2016
+``` -->
+
+```
+# single-gpu testing
+python test.py --checkpoint /path/to/checkpoint \
+--config /path/to/config_file \
+--eval mIoU \
+[--out ${RESULT_FILE}] [--show] \
+--aug-test \ # for multi-scale flip aug
+
+# multi-gpu testing (4 gpus, 1 sample per gpu)
+python -m torch.distributed.launch --nproc_per_node=4 --master_port=29500 \
+test.py  --launcher pytorch --eval mIoU \
+--config_file /path/to/config_file \
+--checkpoint /path/to/checkpoint \
+--aug-test \ # for multi-scale flip aug
+```
+
+## Results
+We report results on validation sets.
+
+| Backbone | Crop Size | Batch Size | Dataset | Lr schd | Mem(GB) | mIoU(ms+flip) | config |
+| :------: | :-----: | :-----: | :-----: | :-----: | :-----: | :-----: | :-----: | 
+| Res-50 | 480x480 | 16 | ADE20K | 160K | 7.0G | 46.1 | [config](https://github.com/amazon-research/unified-ept/configs/res50_unept_ade20k.py) |
+| DeiT | 480x480 | 16 | ADE20K | 160K | 8.5G | 50.5 | [config](https://github.com/amazon-research/unified-ept/configs/deit_unept_ade20k.py) |
+| DeiT | 480x480 | 16 | PASCAL-Context | 160K | 8.5G | 55.2 | [config](https://github.com/amazon-research/unified-ept/configs/deit_unept_pcontext.py) |
 
-See [CONTRIBUTING](CONTRIBUTING.md#security-issue-notifications) for more information.
 
 ## License
 
 This project is licensed under the Apache-2.0 License.
 
+## Citation
+
+If you use this code and models for your research, please consider citing:
+
+```
+@article{zhu2021unified,
+  title={A Unified Efficient Pyramid Transformer for Semantic Segmentation},
+  author={Zhu, Fangrui and Zhu, Yi and Zhang, Li and Wu, Chongruo and Fu, Yanwei and Li, Mu},
+  journal={arXiv preprint arXiv:2107.14209},
+  year={2021}
+}
+```
+
+## Acknowledgment
+
+We thank the authors and contributors of [MMCV](https://mmcv.readthedocs.io/en/latest/), [MMSegmentation](https://github.com/open-mmlab/mmsegmentation/blob/master/docs/install.md), [timm](https://pypi.org/project/timm/) and [Deformable DETR](https://github.com/fundamentalvision/Deformable-DETR).
+<!-- ### Using Longformer
```

Comment:
The following content seems to be a conflict during git merge. Please confirm and delete unnecessary content.